src/app/sc2/utils/match_utils.py

```diff
@@ -0,0 +1,15 @@
+""" utils for match """
+from app.sc2.utils.player_utils import prettify_name
+
+
+def get_vs_string_from_match(match):
```

Comment:
If you are only using this stuff for slack I would just throw it in the slack file.